initfiles/bin/init_thorslave

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+################################################################################
+#    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+################################################################################
+
+ip=$1
+master=$2
+masterport=$3
+logpth=$4
+instancedir=$5
+deploydir=$6
+hpcc_compname=$7
+hpcc_setenv=$8
+logredirect=$9
+cmd=${10}
+
+source ${hpcc_setenv}
+
+stop_slaves()
+{
+    local slavename=thorslave_${hpcc_compname}
```

Comment:
Moved to outside the function, and used slavename instead of thorslave_${hpcc_compname} where applicable.